src/schema/v2/fields/money.ts

```diff
@@ -137,6 +137,18 @@ export const Money = new GraphQLObjectType<any, ResolverContext>({
         return cents / factor
       },
     },
+    toUSD: {
+      type: new GraphQLNonNull(GraphQLFloat),
+      description: "An amount of money converted to USD major units (dollars).",
+      resolve: async ({ cents, currency }, {}, { exchangeRatesLoader }) => {
```

Comment:
This resolver might have a special case for amounts already denominated in `USD` to avoid a fetch for the exchange rates.

Reply:
Good call, incorporated into the new implementation